src/coreclr/debug/di/module.cpp

```diff
@@ -2758,18 +2758,20 @@ HRESULT CordbModule::GetJITCompilerFlags(DWORD *pdwFlags )
 
 HRESULT CordbModule::IsMappedLayout(BOOL *isMapped)
 {
+    PUBLIC_API_ENTRY(this);
     VALIDATE_POINTER_TO_OBJECT(isMapped, BOOL*);
     FAIL_IF_NEUTERED(this);
 
     HRESULT hr = S_OK;
     CordbProcess *pProcess = GetProcess();
 
     ATT_REQUIRE_STOPPED_MAY_FAIL(pProcess);
-    PUBLIC_API_BEGIN(pProcess);
+
+    EX_TRY
     {
         hr = pProcess->GetDAC()->IsModuleMapped(m_vmModule, isMapped);
     }
-    PUBLIC_API_END(hr);
+    EX_CATCH_HRESULT(hr);
 
```

Comment:
@hoyosjs This is @chuckries proposed fix to `CordbModule::IsMappedLayout`

Reply:
No opposition but it was unclear how this fix relates to the data breakpoints issue? I think we should either clarify how it is connected in comments/commit text or fix it in a separate non-servicing PR.